lib/settlement/driip-settlement-contract.js

```diff
@@ -1,24 +1,10 @@
 'use strict';
 
-const ethers = require('ethers');
+const NahmiiContract = require('../contract');
```

Comment:
'../contract': Litt anonymt navn

Reply:
Since pretty much everything is prefixed with nahmii there is no need to repeat that everywhere.. naming by convention.